src/main.py

```diff
@@ -20,7 +20,7 @@
 import discord.errors
 from discord.ext import tasks
 from datetime import datetime, timezone, timedelta
-from birthdays import check_birthdays
+from scripts.birthdays import check_birthdays
```

Comment:
`scripts` are generally intended to be run from the CLI, i.e. `python scripts/script_name.py`. semantically something you import would probably fit in better somewhere under `src/`. and also importing stuff from directories without an `__init__.py` in them can sometimes end up weird IIRC